README.md

```diff
@@ -26,6 +26,7 @@ Where opts is an object with the following properties (all optional):
 * **simhostui** - the directory containing the UI specific files of the simulation host. Defaults to the bundled simulation host files, found in `src/sim-host/ui`.
 * **livereload** - A boolean. Set to false to disable live reload. Defaults to true.
 * **forceprepare** - A boolean. Set to true to force a `cordova prepare` whenever a file changes during live reload. If this is false, the server attempts to simply copy the changed file to the platform rather than doing a `cordova prepare`. Ignored if live reload is disabled. Defaults to false.
+* **xhrproxy** - Boolean indicating if a XHR should go through the server proxy. Defaults to true.
```

Comment:
This is for avoiding CORS. I guess that it would be more clear if instead naming the option `xhrproxy`, we use `corsproxy`.